Gamedata/Bluedog_DB/Parts/Saturn/bluedog_Saturn_S4B_SLA.cfg

```diff
@@ -31,7 +31,7 @@ category = Payload
 subcategory = 0
 title = Sarnus-SIVB-LAM Lander Adapter Module
 manufacturer = Bluedog Design Bureau
-description = This adapter opens in four 'petals' to separate the Kane-11 Service Engine's hidden attach node from a Sarnus stack. Staging the decoupler opens the petals and releases the CSM. The Sina MEM is attached to the payload structure on the inside. Manually decouple the MEM once the transposition and docking manuever is complete.
+description = This 3.35m adapter opens in four 'petals' to separate the Kane-11 Service Engine's hidden attach node from a Sarnus stack. Staging the decoupler opens the petals and releases the CSM. The Sina MEM is attached to the payload structure on the inside. Manually decouple the MEM once the transposition and docking manuever is complete.

```

Comment:
I'm going to merge, just a note for later, 3.35m is wrong. It's a 3.75m to 2.5m adapter. Great job!